app/helpers/mustache_helper/translator.rb

```diff
@@ -0,0 +1,40 @@
+module MustacheHelper
+  # @todo move into extension of Mustache renderer, detecting template variables
+  #   starting with "i18n."?
+  class Translator
+    include ActionView::Helpers::TagHelper
+
+    def initialize(scope, data = {}, parent_keys = [])
+      @scope = scope
+      @data = data
+      @parent_keys = parent_keys
+    end
+
+    def [](key)
```

Comment:
Assignment Branch Condition size for [] is too high. [25.5/15]<br>Method has too many lines. [11/10]